quattor/blockdevices.pan

```diff
@@ -1,41 +1,27 @@
-################################################################################
-# This is 'namespaces/standard/quattor/blockdevices.tpl', a pan-templates's file
-################################################################################
-#
-# VERSION:    3.2.9, 25/11/09 16:16
-# AUTHOR:     Martin Bock
-# MAINTAINER: Marco Emilio Poleggi <Marco.Emilio.Poleggi@cern.ch>, German Cancio <German.Cancio.Melia@cern.ch>, Michel Jouvin <jouvin@lal.in2p3.fr>
-# LICENSE:    http://cern.ch/eu-datagrid/license.html
-#
-################################################################################
-# Coding style: emulate <TAB> characters with 4 spaces, thanks!
-################################################################################
-#
-# type definition for block devices.
-#
-#
-#
-# DESCRIPTION: @DESCRIPTION@
-
-#
-############################################################
+@{
+    Type definitions for block devices.
```

Comment:
reading http://quattor-pan.readthedocs.org/en/latest/pan-book/pan-book.html?highlight=annotation, `desc` seems to be used for functions (and require the `@documentation{}`).